src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -39,6 +39,7 @@ namespace Microsoft.Terminal.TerminalControl
         Microsoft.Terminal.TerminalControl.IKeyBindings KeyBindings;
 
         Boolean CopyOnSelect;
+        Boolean InputServiceWarning;

```

Comment:
This doesn't actually need to be here. We don't need this setting down in the control layer, so we can omit it here.